Web/www/pid.json

```
@@ -0,0 +1,20 @@
+
+[
+    {
+        "p": "1",
+        "i": "2",
+        "d": "3"
+    },
+
+    {
+        "p": "1",
+        "i": "0",
+        "d": "1"
+    },
+
+    {
+        "p": "0",
+        "i": "1",
+        "d": "0"
+    }
+]
```

```suggestion
[
  {
    "name": "pitch",
    "data": {
      "p": 1,
      "i": 2,
      "d": 3
    }
  },
  {
    "name": "roll",
    "data": {
      "p": 1,
      "i": 0,
      "d": 1
    }
  },
  {
    "name": "yaw",
    "data": {
      "p": 0,
      "i": 1,
      "d": 0
    }
  }
]
```
Would it be possible to change the structure of the data to something like this so that it's easier to see what PID values are associated with what controller?

---

Fair enough 👍  I didn't have a clear structure before because I was trying to do some looping stuff 😬